interactivity/ai-powered-insights-overview.md

```diff
@@ -0,0 +1,46 @@
+
+>tip For a working example of this functionality, check the [AI Insights Report Demo](https://demos.telerik.com/reporting/ai-insights).
+
+![The UI of the AI system after configuration.](images/angular-report-viewer-with-ai-insights.png)
+
+## How Does It Work?
+
+To bring the power of Generative AI (GenAI) into reporting workflows, we are introducing an **AI Prompt** dialog that integrates seamlessly in the report viewers. The dialog provides a convenient UI for sending predefined or custom prompts to an external AI model (for example, GPT-5), configured in the Reporting REST Service. The prompts and responses returned from the AI model are displayed in the **Output** panel of the dialog, allowing for easier involvement in the conversation.
+
+The AI conversation maintains context throughout user's interaction with a specific report. All previous questions and responses are preserved and sent to the AI model as context, enabling more coherent and contextually relevant conversations. However, this context is automatically cleared when report parameters are changed or when navigating to a different report, ensuring that each report session starts with a fresh conversation thread.
+
+The feature is supported by all [web report viewers]({%slug telerikreporting/using-reports-in-applications/display-reports-in-applications/web-application/html5-report-viewer/overview%}) and by the [WPF Report Viewer]({%slug telerikreporting/using-reports-in-applications/display-reports-in-applications/wpf-application/overview%}) connected to a remote Reporting REST Service.
+
+### Key Features
+
+- Retrieval-Augmented Generation (RAG)—When enabled, the AI insights feature uses an algorithm that filters out the irrelevant report data, producing more accurate responses with reduced token usage.
+  
+- Predefined Summary Prompts—Users can choose from a set of predefined prompts tailored for common tasks like summarization, explanation, and data insights—boosting efficiency with minimal effort.
+
+- Custom AI Prompts—Besides the predefined prompts, users can create custom prompts to ask more specific queries.
+
+	<img src="images/predefined-prompts.png" style="border: 1px solid lightgray; width: 500px" alt="Image of the Prompt UI" />
+
+- End-User Consent for Data Sharing—To ensure transparency and compliance, the AI Prompt requests explicit consent from users before sending their prompts to the AI models.
+
+	<img src="images/user-consent.png" style="border: 1px solid lightgray; width: 500px" alt="User Consent for AI Summaries" />
+
```

Comment:
You may want to add a section called "Configuring the AI-Powered Insights" (or something similar). In this section, you could describe, on a very high level, the most straightforward configuration by providing links to the relevant articles describing the configuration in detail.

Adding this section could be vital, as for me at least, it wasn't immediately clear which article I needed to review next in order to configure the insights.